extensions/http(s)/test/ace/http/request_test.exs

```diff
@@ -49,6 +49,15 @@ defmodule Ace.HTTP.RequestTest do
     assert upload.type == "text/plain"
   end
 
+  test "handles absoluteURI", %{port: port} do
+    request = """
+    GET http://www.raxx.com/ HTTP/1.1
+    """
```

Comment:
To make sure this is a valid request there needs to be a full empty line after headers.